camel/storages/graph_storages/nebula_graph.py

```diff
@@ -188,7 +188,8 @@ def ensure_edge_type_exists(
         self,
         edge_type: str,
     ) -> None:
-        r"""Ensures that a specified edge type exists in the NebulaGraph
+        r"""
+        Ensures that a specified edge type exists in the NebulaGraph
```

Comment:
there's no need to change this part as it's camel's docstring standard, refer to : https://github.com/camel-ai/camel/blob/master/CONTRIBUTING.md#guideline-for-writing-docstrings

```suggestion
        r"""Ensures that a specified edge type exists in the NebulaGraph
```